mitodl/mitxpro
# Added RedBeat to handle task scheduling

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #1458 

#### What's this PR do?
Adds [redbeat](https://github.com/sibson/redbeat) as an app dependency and configures the app to use it. Redbeat uses Redis as a backend for task scheduling instead of the file system, so 

#### How should this be manually tested?
`docker-compose build`, `docker-compose up`, and make sure that scheduled tasks are still being executed. The way I did this was to set `RETRY_FAILED_EDX_ENROLLMENT_FREQUENCY=10` in my .env so the `retry-failed-edx-enrollments` would be run every 10 seconds.

#### Any background context you want to provide?
OVS also uses this library for task scheduling
